test/integration/arch/README.md

```diff
@@ -0,0 +1,50 @@
+# InSpec Profile: `default`
```

Comment:
The actual suite name needs to be used throughout this file, will be resolved when commit is added here:

https://github.com/myii/jetbrains-pycharm-formula/commit/a3bac46edc0678d4436b425e3929d5c04aea72f6#diff-19a94498bc41afb14c871deec9e4b473L1-R43